sdk/src/adminClient.ts

```diff
@@ -3528,4 +3531,36 @@ export class AdminClient extends DriftClient {
 			},
 		});
 	}
+
+	public async initializePythPullOracle(
+		feedId: string
+	): Promise<TransactionSignature> {
+		const initializePythPullOracleIx = await this.getInitializePythPullOracleIx(
+			feedId
+		);
+		const tx = await this.buildTransaction(initializePythPullOracleIx);
+		const { txSig } = await this.sendTransaction(tx, [], this.opts);
+
+		return txSig;
+	}
+
+	public async getInitializePythPullOracleIx(
+		feedId: string
+	): Promise<TransactionInstruction> {
+		if (feedId.startsWith('0x')) {
```

Comment:
we reusue this multple places, should we just have a single getFeedIdBuffer fn that handles this?